src/coord/src/session.rs

```diff
@@ -356,8 +380,10 @@ pub enum TransactionOps {
     /// The transaction has been initiated, but no statement has yet been executed
     /// in it.
     None,
-    /// This transaction has had a read (`SELECT`, `TAIL`) and must only do other reads.
-    Reads,
+    /// This transaction has had a peek (`SELECT`, `TAIL`) and must only do other peeks.
+    Peeks(Timestamp),
```

Comment:
why did we rename from Read -> Peek?

Reply:
Because that's the dataflow operation that occurs? I don't remember why I did this but I think it's fine?